app/models/taxon_name.rb

```diff
@@ -16,6 +16,8 @@ class TaxonName < ActiveRecord::Base
   validates_format_of :lexicon, with: /\A[^\/,]+\z/, message: :should_not_contain_commas_or_slashes, allow_blank: true
   validate :species_common_name_cannot_match_taxon_name
   validate :valid_scientific_name_must_match_taxon_name
+  validate :english_lexicon_if_exists
+  validate :latin_character_set
```

Comment:
IMO, this is probably taking it a bit too far. People add names not only so they can "use" them on iNat when searching for taxa but also so they can be listed on the taxon page where people can learn about them or at least be made aware they exist. While I want to avoid duplication, I don't want to suppress the addition and correct spelling of languages that we do not currently support for translation. There are vanishingly few such lexicons that have been added, but I want to avoid a situation where someone wants to see their language represented on the website and can't do it properly b/c we force them to use Latin characters.

Reply:
Hmm, yeah definitely don't want to prohibit anyone's language from being transliterated into English properly. This is an extended latin character set with all sorts of diacriticals, but still I won't pretend to know if the thousands of languages can be represented with even that. I'll pull this.